data/sql/updates/pending_db_world/rev_1698463311896667349.sql

```diff
@@ -0,0 +1,26 @@
+-- Thekal (14509)
+-- Update SAY_AGGRO to include emote ONESHOT_ROAR
+-- Update comments
+DELETE FROM `creature_text` WHERE `CreatureID`=14509;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(14509, 0, 0, 'Shirvallah, fill me with your RAGE!', 14, 0, 100, 15, 0, 8419, 10455, 0, 'High Priest Thekal - BOSS_SAY_AGGRO'),
+(14509, 1, 0, 'Hakkar binds me no more!  Peace at last!', 14, 0, 100, 0, 0, 8424, 10451, 0, 'High Priest Thekal - BOSS_SAY_DEATH'),
+(14509, 2, 0, '%s dies.', 16, 0, 100, 0, 0, 0, 8251, 3, 'High Priest Thekal - BOSS_EMOTE_DIES');
+
+-- Thekal (14509)
+-- Remove static idle (talking) emote
+DELETE FROM `creature_addon` WHERE (`guid` IN (49310));
```

Comment:
Should this and the following INSERT/DELETEs be UPDATE? I read the [SQL Standard](https://www.azerothcore.org/wiki/sql-standards#insert-amp-delete) but it doesn't necessarily address when each method should be used. Happy to update them if needed.

Reply:
It's kind of a feeling for it. Say you update positions for 100 creatures, well then UPDATE would not be the most efficient. But here, if the entry already exist in creature_addon, then UPDATE would be to prefer

Reply:
I changed the SQL to use UPDATEs as requested.